src/components/Rating.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function Rating(props) {
+    const {children} = props
+    function stars (children) {
+        if(children >= 4.5) return "★★★★★";
+        else if (children >= 3.5) return "★★★★☆"
+        else if(children >= 2.5) return "★★★☆☆"
+        else if(children >= 1.5) return "★★☆☆☆"
+        else if(children >= 0.5) return "★☆☆☆☆"
+        else if(children === 0) return "☆☆☆☆☆"
```

Comment:
For it to work using this logic, you just need to take one = out, like:

` else if(children == 0) return "☆☆☆☆☆"`